src/types/leads.ts

```diff
@@ -2,27 +2,29 @@
 
 import { CustomFieldInput, CustomFieldData } from './custom-fields';
 import { PaginatorInfo } from './paginator';
-
-interface Contact {
-  value: string;
-  contacts_types_id?: number;
-  weight?: number;
-  type?: Type;
-}
-
-interface Address {
-  address: string;
-  city: string;
-}
-
-interface People {
-  name?: string;
-  firstname?: string;
-  lastname?: string;
-  contacts: Contact[];
-  address?: Address[];
-  custom_fields?: CustomFieldInput[];
-}
+import {PeopleInterface} from './index';
+// interface Contact {
+//   value: string;
+//   contacts_types_id?: number;
```

Comment:
Question: why is this commented on? if we are not using it we can remove these comments

Reply:
Because I want the confirmation from @jerlyrosa and @lizandro-mc